benefits/core/templates/core/help.html

```diff
@@ -152,8 +143,8 @@ <h2 class="h2-sm pt-4 pt-lg-8" id="questions">{% translate "Questions?" %}</h2>
       </div>
     </div>
 
-    <div class="row pt-4 pt-lg-8">
-      <div class="col-lg-2 offset-lg-10 col-8 offset-2">
+    <div class="row justify-content-center pt-4 pt-lg-8">
+      <div class="col-12 col-lg-6">
         {% translate "Go Back" as button_text %}
         {% include "core/includes/button--origin.html" with button_text=button_text %}
       </div>
```

Comment:
I tried using the `call-to-action` or `call-to-action-button` blocks instead, but I can't b/c I'm overriding the parent `main-content` block entirely. I'm okay with it though, b/c this page is a different alignment and design from the rest of the pages... so it feels appropriate to override the `base` layout.